vm/ubpf_jit_x86_64.c

```diff
@@ -27,13 +27,47 @@
 #include "ubpf_int.h"
 #include "ubpf_jit_x86_64.h"
 
+#if !defined(_countof)
+    #define _countof(_Array) (sizeof(_Array) / sizeof(_Array[0]))
```

Comment:
```suggestion
    #define _countof(array) (sizeof(array) / sizeof(array[0]))
```

Reply:
Removed this as this is a Windows quirk (having _countof not defined for kernel mode).

Reply:
Actually, got that backwards. _countof is a Windows specific macro.